Ceramic-SelfID.md

```diff
@@ -441,7 +441,7 @@ return (
       onChange={(e) => setName(e.target.value)}
       className={styles.mt2}
     />
-    <button onClick={() => updateRecordName(name)}>Update</button>
+    <button className={styles.button} onClick={() => updateRecordName(name)}>Update</button>
```

Comment:
Could you send a screenshot here of how that looks please?

Reply:
Please find the screenshot below

![LearnWeb3Progress64](https://user-images.githubusercontent.com/14857229/236408208-8459bbb2-bce1-4848-b29f-ff214e1d5e4e.png)